taqueria-plugin-flextesa/proxy.ts

```diff
@@ -19,31 +31,66 @@ interface Opts extends RequestArgs.ProxyRequestArgs {
 
 const getDockerImage = (opts: Opts) => `ghcr.io/ecadlabs/taqueria-flextesa:${opts.setVersion}-${opts.setBuild}`;
 
-const getContainerName = (sandboxName: string, parsedArgs: Opts) => {
-	return `taqueria-${parsedArgs.env}-${sandboxName}`;
+const getUniqueSandboxname = async (sandboxName: string, projectDir: string) => {
+	const hash = await stringToSHA256(projectDir);
+	return `${sandboxName}-${hash}`;
+};
```

Comment:
I should update a related function in TVsCE. I'll do it in #1012

Reply:
Ohhh didn't know it's in VSCode too. Btw `getContainerName()` uses this @AlirezaHaghshenas

Reply:
```typescript
const getUniqueSandboxname = async (sandboxName: string, projectDir: string) => {
	const hash = await stringToSHA256(projectDir);
	return `${sandboxName}-${hash}`;
};

const getContainerName = async (sandboxName: string, parsedArgs: Opts) => {
	const uniqueSandboxName = await getUniqueSandboxname(sandboxName, parsedArgs.projectDir);
	return `taqueria-${parsedArgs.env}-${uniqueSandboxName}`;
};
```

Reply:
Nice decision to include hash of project path in container name.

I don't see it being persisted anywhere in the config. If at a later stage a change causes this name to be persisted, we will have the problem that different people working on the same repository will constantly change each others' container names, because project path is different on their machines.

No such problem for now.

Reply:
Unfortunately, I cannot use these functions directly, as TVsCE code should not be dependent on a plugin's code.

For the moment, I duplicate this logic, and maintain it if changes are made to these functions.

@mweichert  might have a suggestion to stay DRY without adding dependency.